pom.xml

```diff
@@ -743,14 +726,14 @@
           </configuration>
         </plugin>
         <plugin>
-          <groupId>org.codehaus.mojo</groupId>
-          <artifactId>findbugs-maven-plugin</artifactId>
+          <groupId>com.github.spotbugs</groupId>
+          <artifactId>spotbugs-maven-plugin</artifactId>
           <version>3.0.5</version>
```

Comment:
Should use the `${spotbugs.plugin.version}` here as well.

Reply:
https://github.com/cerner/beadledom/commit/66d28e32652a3d9b720293986ee67e0fead66a99?diff=unified#diff-600376dffeb79835ede4a0b285078036L731